cmd/ocm/list/version/cmd.go

```diff
@@ -55,6 +56,13 @@ func init() {
 		"stable",
 		"List only versions from the specified channel group",
 	)
+
+	fs.StringVar(
+		&args.gcpMarketplace,
+		"gcp-marketplace",
```

Comment:
should this be `marketplace-gcp` instead?

Reply:
Changed